src/localStorage.js

```diff
@@ -0,0 +1,20 @@
+export const loadState = () => {
+  try {
+    const serializedState = localStorage.getItem('state');
```

Comment:
I think you should prepend `localStorage` with `window` so there is no ambiguity where this variable comes from. Also if declare a variable in this module it will override the one that comes from `window`.